docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+
+### Create an empty project
+
+Delete all the files in the `tutorial` folder and run the following command to create an empty project:
+
+```sh
+dlt project init
+```
+
+This will create a project without any sources, destinations, datasets, or pipelines; the project will be named after the folder.
+
+### Add all entities
+
+Now we can add all of our entities individually. This way, we can also give them their own names, which will be useful when having multiple destinations of the same type, for example.
+
+Add a source:
+
+```sh
+# add a new arrow source called "my_arrow_source"
+dlt source my_arrow_source add arrow
+```
+
+Add a destination:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+# this will also create a new dataset called "my_duckdb_destination_dataset"
+dlt destination my_duckdb_destination add duckdb
+```
+
+If you want to create a dataset automatically, you can use the `--dataset-name` flag:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+dlt destination my_duckdb_destination add duckdb --dataset-name my_duckdb_destination_dataset
+```
+
+Now we can add a pipeline that uses the source and destination we just added:
+
+```sh
+# add a new pipeline called "my_pipeline" which loads from my_arrow_source and saves to my_duckdb_destination
+# we select the my_duckdb_destination_dataset with the optional flag
+dlt pipeline my_pipeline add my_arrow_source my_duckdb_destination --dataset_name my_duckdb_destination_dataset
+```
+
+### Adding the core source
+
+You can add multiple entities using CLI commands. Let's add another source - this time, a core source such as a
+([REST API](../../dlt-ecosystem/verified-sources/rest_api/index.md), [SQL database](../../dlt-ecosystem/verified-sources/sql_database/index.md), [filesystem](../../dlt-ecosystem/verified-sources/filesystem/index.md)).
+
+Run the following command to add an SQL database source named `sql_db_1`:
+```sh
+# add a new arrow source called "sql_db_1"
+dlt source sql_db_1 add sql_database
+```
+
+This will add the new source to your `dlt.yml` file:
+
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+  sql_db_1:
+    type: sql_database
+```
+
+The corresponding credential placeholders will be added to `.dlt/secrets.toml`, but you can also define them in `dlt.yml`.
+```toml
+[sources.sql_db_1]
+table_names = ["family", "clan"]
+
+[sources.sql_db_1.credentials]
+drivername = "mysql+pymysql"
+database = "Rfam"
+username = "rfamro"
+host = "mysql-rfam-public.ebi.ac.uk"
+port = 4497
+```
+
+## Configuration and profiles
```

Comment:
Generally I think we might have to have a stronger story about why to use profiles (in the section above), but that's again probably for a future change.